python/ray/train/constants.py

```diff
@@ -67,6 +67,15 @@
 TRAIN_ENABLE_WORKER_SPREAD_ENV = "TRAIN_ENABLE_WORKER_SPREAD"
 
 
+TRAIN_ENV_VARS = {
```

Comment:
Should we do the same for AIR/Tune?

Reply:
Yes, good idea. Any ideas on **how to add a note about adding a key for any new environment variables that get introduced?**

Reply:
Nothing obvious comes to mind. Having this collocated with the other environment variables will be a good "best effort" starting point, so developers can follow by example.